mct/m_SPMDutils.F90

```diff
@@ -24,6 +24,16 @@
 !
 ! !INTERFACE:
 
+! Code added as a work around for poor performance or incorrect functionality
+! in MPI libraries
+!
+#ifndef _USE_MPI_RSEND
```

Comment:
Never mind - I did it manually, via another commit. Should be ready to go. Still curious ... how would I set the _USE_MPI_RSEND token when building MCT in ACME? Thanks.